src/components/change-requests/new-change-request/new-change-request-page/new-change-request-page.tsx

```diff
@@ -3,38 +3,239 @@
 import ActivationFormFields from './activation-form-fields/activation-form-fields'
 import StageGateFormFields from './stage-gate-form-fileds/stage-gate-form-fields'
+import LoadingIndicator from '../../../shared/loading-indicator/loading-indicator';
+import ErrorPage from '../../../shared/error-page/error-page';
 import styles from './new-change-request-page.module.css';
+import { wbsPipe } from '../../../../shared/pipes';
+import { CR_Type, Scope_CR_Why_Type } from '@prisma/client';
+import { useCreateChangeRequest } from '../../../../services/change-requests.hooks';
+import { useHistory } from 'react-router-dom';
+import { useAuth } from '../../../../services/auth.hooks';
+import { NewActivationChangeRequestPayload, NewStageRequestChangeRequestPayload, NewStandardChangeRequestPayload } from 'utils';
+import { NewChangeRequestPayload } from 'utils/src';
+
+interface FormData {
+  projectWBS: number,
+  workPackageWBS: number,
+  type: CR_Type,
+
+  what: string,
+  scopeImpact: string,
+  timelineImpact: number,
+  budgetImpact: number,
+  estimation_error: boolean,
+  school_work: boolean,
+  manufacturing_issues: boolean,
+  rules_compliance: boolean,
+  other_project: boolean,
+  other: boolean,
+  other_project_explain: string,
+  other_explain: string,
+
+  projectLeadId: number,
+  projectManagerId: number,
+  startDate: Date,
+  confirmDetails: boolean,
+
+  leftoverBudget: number,
+  confirmDone: boolean,
+}
+
+interface WhyResponse {
+  type: Scope_CR_Why_Type,
+  explain: string
+}
 
 const NewChangeRequestPage: React.FC = () => {
-  const submitHandler = () => {
-    alert('submitted');
+  const createChangeRequest = useCreateChangeRequest();
+  const auth = useAuth();
+  const history = useHistory();
+
+  const submitHandler = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    const { type, projectWBS, workPackageWBS } = formData;
+    const wbsId = (workPackageWBS === -1) ? projectWBS : workPackageWBS;
+
+    const { user } = auth;
+    const { userId } = user!;
+
+    let req: NewStandardChangeRequestPayload | NewStageRequestChangeRequestPayload | NewActivationChangeRequestPayload | undefined;
+
+    if (type === CR_Type.ACTIVATION) {
+      req = { 
+        projectLeadId: formData.projectLeadId,
+        projectManagerId: formData.projectManagerId,
+        startDate: formData.startDate.toJSON(),
+        confirmDetails: formData.confirmDetails
+      };
+    }
+    else if (type === CR_Type.STAGE_GATE) {
+      req = { leftoverBudget: formData.leftoverBudget, confirmDone: formData.confirmDone };
+    }
+    else {
+      const { estimation_error, 
+        school_work, 
+        manufacturing_issues, 
+        rules_compliance,
+        other_project,
+        other,
+        other_project_explain,
+        other_explain
+      } = formData;
+
+      const whyOpts: WhyResponse[] = [];
+
+      if (estimation_error) whyOpts.push({'type': Scope_CR_Why_Type.ESTIMATION, explain: ""});
```

Comment:
definitely want to file a ticket to make this more flexible because we need an additional one named "DESIGN" for MVP launch, and may want more in the future